src/core.js

```diff
@@ -441,7 +441,6 @@ $.extend( $.validator, {
 			if ( errors ) {
 				// add items to error list and map
 				$.extend( this.errorMap, errors );
-				this.errorList = [];
```

Comment:
Yes, you're right. That's why I said `I'm not sure`.

I think we should only extend `this.errorList`, so we have to override old errors and add new ones. Right now I'm thinking of doing:
```js
this.errorList = $.grep( this.errorList, function( error ) {
	return !( error.element.name in errors );
} );
```
in place of set an empty array to `this.errorList`

What do you think ?